### ISSUE TITLE
AdGuard Home version remains 0.102.0 in add-on version 2.5

### ISSUE THREAD

**Issue by berichta** [hassio-addons/addon-adguard-home#78]
# Problem/Motivation

Addon update didn't update the AdGuard Home version as stated in the release notes.

## Expected behavior

AdGuard Home version should be 0.103.3

## Actual behavior

AdGuard Home version is v0.102.0

## Steps to reproduce

Update AdGuard Home addon from previous version to 2.5.0

## Logs

```
Add-on: AdGuard Home
 Network-wide ads & trackers blocking DNS server
-----------------------------------------------------------
 Add-on version: 2.5.0
 You are running the latest version of this add-on.
 System: null  (armv7 / raspberrypi3)
 Home Assistant Core: 0.113.2
 Home Assistant Supervisor: 229
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] adguard.sh: executing... 
[cont-init.d] adguard.sh: exited 0.
[cont-init.d] discovery.sh: executing... 
[08:45:35] INFO: Successfully send discovery information to Home Assistant.
[cont-init.d] discovery.sh: exited 0.
[cont-init.d] nginx.sh: executing... 
[cont-init.d] nginx.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[08:45:38] INFO: Starting AdGuard Home server...
2020/07/29 08:45:38 [info] AdGuard Home, version v0.102.0, channel release, arch linux arm v6
2020/07/29 08:45:38 [info] Initializing auth module: /data/adguard/data/sessions.db
2020/07/29 08:45:38 [info] Auth: initialized.  users:0  sessions:0
2020/07/29 08:45:38 [info] Initialize web module
2020/07/29 08:45:38 [info] Bootstrap 0: 1.1.1.1:53
2020/07/29 08:45:38 [info] Bootstrap 1: 1.0.0.1:53
2020/07/29 08:45:38 [info] Upstream 0: tls://dns.quad9.net:853
2020/07/29 08:45:38 [info] Upstream 1: https://cloudflare-dns.com:443/dns-query
2020/07/29 08:45:39 [info] Go to http://127.0.0.1:45158
[08:45:39] INFO: Starting NGinx...
```

**Comment by stevehop** [comment]
Just updated and working ok here:
2020/07/29 11:32:06 [info] AdGuard Home, version 0.103.3, channel release, arch linux amd64
2020/07/29 11:32:06 [info] Initializing auth module: /data/adguard/data/sessions.db
2020/07/29 11:32:06 [info] Auth: initialized.  users:0  sessions:0
2020/07/29 11:32:06 [info] number of certs - 2
2020/07/29 11:32:06 [info] got an intermediate cert
2020/07/29 11:32:06 [info] Initialize web module

**Comment by Poeschl** [comment]
Got the 0.102.0 version and even after reinstall and re-pulling the 2.5.0 image, AdGuard Home remains on the old version

**Comment by AlecRust** [comment]
The [latest release](https://github.com/hassio-addons/addon-adguard-home/releases/tag/v2.5.0) of this add-on requires Home Assistant minimum version 0.113.2.

**Comment by berichta** [comment]
@Poeschl : Please can you check your system? Might be useful to understand if you are running on armv7 as well. Furthermore, I can confirm that reinstalling the addon didn't resolve the issue.

@AlecRust : The minimum version shouldn't have an effect on this issue. Going through the provided log you will notice that I was on 0.113.2. Without, you wouldn't be able to install version 2.5 of the addon at all.

**Comment by Poeschl** [comment]
My system info:

```
arch | armv7l
chassis | embedded
dev | false
docker | true
docker_version | 19.03.8
hassio | true
host_os | HassOS 4.11
installation_type | Home Assistant OS
os_name | Linux
os_version | 4.19.127-v7l
python_version | 3.8.3
supervisor | 229
timezone | Europe/Berlin
version | 0.113.2
virtualenv | false
```

**Comment by kwertie01** [comment]
I have the same issue.
Today I upgraded HA successfully to version 0.113.2 and upgraded AdGuard from v2.4.2 to v2.5.0
sadly the AdGuard version stayed at v0.102.0


Add-on: AdGuard Home
Network-wide ads & trackers blocking DNS server
 Add-on version: 2.5.0
 You are running the latest version of this add-on.
 System: HassOS 4.11  (armv7 / raspberrypi3)
 Home Assistant Core: 0.113.2
 Home Assistant Supervisor: 229
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to INFO
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] adguard.sh: executing... 
[cont-init.d] adguard.sh: exited 0.
[cont-init.d] discovery.sh: executing... 
[16:31:14] INFO: Successfully send discovery information to Home Assistant.
[cont-init.d] discovery.sh: exited 0.
[cont-init.d] nginx.sh: executing... 
[cont-init.d] nginx.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[16:31:16] INFO: Starting AdGuard Home server...
2020/07/29 16:31:16 [info] AdGuard Home, version v0.102.0, channel release, arch linux arm v6
2020/07/29 16:31:16 [info] DHCP: loaded 47 (47) leases from DB
2020/07/29 16:31:16 [info] Initializing auth module: /data/adguard/data/sessions.db
2020/07/29 16:31:16 [info] Auth: initialized.  users:0  sessions:0
2020/07/29 16:31:16 [info] Initialize web module
2020/07/29 16:31:16 [info] Bootstrap 0: 1.1.1.1:53
2020/07/29 16:31:16 [info] Upstream 0: https://dns10.quad9.net:443/dns-query
2020/07/29 16:31:16 [info] Upstream 1: 84.116.46.20:53
2020/07/29 16:31:16 [info] Upstream 2: 84.116.46.21:53
2020/07/29 16:31:16 [info] Upstream 3: 8.8.8.8:53
2020/07/29 16:31:17 [info] DHCP: listening on 0.0.0.0:67
2020/07/29 16:31:17 [info] Go to http://127.0.0.1:45158
[16:31:18] INFO: Starting NGinx...
nginx: [alert] detected a LuaJIT version which is not OpenResty's; many optimizations will be disabled and performance will be compromised (see https://github.com/openresty/luajit2 for OpenResty's LuaJIT or, even better, consider using the OpenResty releases from https://openresty.org/en/download.html)
2020/07/29 16:31:27 [info] Starting the DNS proxy server
2020/07/29 16:31:27 [info] Ratelimit is enabled and set to 20 rps
2020/07/29 16:31:27 [info] The server is configured to refuse ANY requests
2020/07/29 16:31:27 [info] DNS cache is enabled
2020/07/29 16:31:27 [info] Creating the UDP server socket
2020/07/29 16:31:27 [info] Listening to udp://[::]:53
2020/07/29 16:31:27 [info] Creating the TCP server socket
2020/07/29 16:31:27 [info] Listening to tcp://[::]:53
2020/07/29 16:31:27 [info] Entering the tcp listener loop on [::]:53
2020/07/29 16:31:27 [info] Entering the UDP listener loop on [::]:53

Info:

--  |  --
-- | --
arch | armv7l
chassis | embedded
dev | false
docker | true
docker_version | 19.03.8
hassio | true
host_os | HassOS 4.11
installation_type | Home Assistant OS
os_name | Linux
os_version | 4.19.127-v7
python_version | 3.8.3
supervisor | 229
timezone | Europe/Amsterdam
version | 0.113.2
virtualenv | false

**Comment by sinclairpaul** [comment]
At this point it appears to be an upstream issues, specifically with the armv7 release.

**Comment by kwertie01** [comment]
@sinclairpaul : Do you have an idea how to go further with this issue? 
Your last message
> At this point it appears to be an upstream issue, specifically with the armv7 release.

doesn't give me a clear direction on how to proceed, to solve this issue.
My AdGuard version is still at v0.102.0
Thanks!

**Comment by sempixel** [comment]
Same here.

* Raspberry 4
* HA 0.114.4
* Addon 2.5.0

Is it related to https://github.com/AdguardTeam/AdGuardHome/issues/1966 ?

**Comment by shreyaskarnik** [comment]
Waiting on AdGuard `0.103.3` any chance #83 can be merged?

**Comment by frenck** [comment]
v2.5.1 has been released

**Comment by regystro** [comment]
Hi. I thought that 2.5.1 would fix my system too, but that's not the case, since I'm stuck at version 2.3.3.
I tried uninstalling add-on, rebooting and reinstalling, but the only available version is 2.3.3.

That's the relevant log:
```
 Add-on version: 2.3.3
 There is an update available for this add-on!
 Latest add-on version: null
 Please consider upgrading as soon as possible.
 System: Raspbian GNU/Linux 10 (buster)  (armv7 / raspberrypi3)
 Home Assistant Core: 0.115.3
 Home Assistant Supervisor: 245
...
INFO: Starting AdGuard Home server...
[info] AdGuard Home, version v0.101.0, channel release, arch linux arm v6%!(EXTRA string=6)
```

It seems it should be downloading armv7 release but is getting armv6.

Thank you.
